tests/test_classifier/test_learning_curve.py

```diff
@@ -0,0 +1,107 @@
+# tests.test_classifier.test_learning_curve
+# Tests for the LearningCurveVisualizer
+#
+# Author:   Jason Keung <jason.s.keung@gmail.com>
+# Created:  Tues May 23 11:45:00 2017 -0400
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_learning_curve.py jason.s.keung@gmail.com $
+
+"""
+Tests for the LearningCurveVisualizer
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+import numpy as np
+
+from ..base import VisualTestCase
+
+from sklearn.svm import LinearSVC
+from sklearn.datasets import load_digits
+from sklearn.model_selection import ShuffleSplit
+from yellowbrick.classifier.learning_curve import LearningCurveVisualizer
+from yellowbrick.classifier.learning_curve import learning_curve_plot
+
+##########################################################################
+## LearningCurveTests Test Cases
+##########################################################################
+
+class LearningCurveTests(VisualTestCase):
+
+    def __init__(self, *args, **kwargs):
```

Comment:
With test cases we use `setUp()` and `tearDown()` - `setUp` is run before every test and `tearDown` run after to make sure that each test runs in a clean state.